public/app/features/alerting/unified/utils/routeTree.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Various helper functions to modify (immutably) the route tree, aka "notification policies"
+ *
+ * make sure to omit the "id" because Prometheus / Loki / Mimir will reject the payload
```

Comment:
looks like this comment shouldn't be there

Reply:
Yea good catch, I've added this comment in other places where this is more relevant 👍